src/polyswarmd/utils.py

```diff
@@ -31,12 +36,12 @@ def bounty_to_dict(bounty):
         'assigned_arbiter': bounty[6],
         'quorum_reached': bounty[7],
         'quorum_reached_block': bounty[8],
-        'quorum_mask': int_to_bool_list(bounty[9]),
+        'quorum_mask': safe_int_to_bool_list(bounty[9], retval['num_artifacts']),
     }
-    if len(bounty) > 10:
+    if bounty_has_voters_and_verdicts:
         retval['bloom'] = uint256_list_to_hex_string(bounty[10])
         retval['voters'] = bounty[11]
-        retval['verdicts'] = int_to_bool_list(bounty[12])
+        retval['verdicts'] = safe_int_to_bool_list(bounty[12], retval['num_artifacts'])
```

Comment:
Not specific to this PR really, but bounty[12] is an array of bool lists, right?

Should it be
```python
retval['verdicts'] = [safe_int_to_bool_list(b, retval['num_artifacts']) for b in bounty[12]]
```

Reply:
Yes, thanks!